database/migrations/1751969302203_convert_cache_ref_to_mobile_configs_table.ts

```diff
@@ -0,0 +1,24 @@
+import { BaseSchema } from "@adonisjs/lucid/schema";
+
+export default class extends BaseSchema {
+  protected tableName = "mobile_configs";
+  protected oldTableName = "cache_reference_numbers";
+
+  async up() {
+    this.schema.raw(
+      `ALTER TABLE ${this.oldTableName} RENAME TO ${this.tableName}`,
+    );
```

Comment:
use `this.schema.renameTable` instead (and make sure to NOT await the returned promise)